assets/src/domain/eventEditor/services/apollo/types.ts

```diff
@@ -19,6 +19,27 @@ export interface EntityEdge<E = Entity, ConnectionTypeName = string> {
 	__typename: ConnectionTypeName;
 }
 
+export interface Event extends Entity, Trashable {
+	capacity: number;
+	description: string;
+	endDate: string;
+	isActive: boolean;
+	isExpired: boolean;
+	isPrimary: boolean;
+	isSoldOut: boolean;
+	isUpcoming: boolean;
+	length: number;
+	order: number;
+	reserved: number;
+	sold: number;
+	startDate: string;
+	status: DatetimeStatus;
+}
```

Comment:
these are datetime properties !?!?!?

Reply:
No, not at all! You can have a look at the GQL schema